samples/apps/copilot-chat-app/webapi/CopilotChat/Skills/ChatSkills/ChatSkill.cs

```diff
@@ -544,6 +544,11 @@ private async Task<string> AcquireExternalInformationAsync(SKContext context, st
 
         var plan = await this._externalInformationSkill.AcquireExternalInformationAsync(userIntent, planContext);
 
+        if (planContext.Variables.TryGetValue("planExecutionTokenUsage", out string? tokenUsage))
```

Comment:
if we change this to `"plannerExecutionTokenUsage"` then we shouldn't need the changes in `GetTokenUsages()`